### PR TITLE
feat(web.exchange): change nok message

### PR DESCRIPTION
**Pull request by benjaminlepine**
ref: MANAGER-12347

| Question         | Answer
| ---------------- | ---
| Branch?          | feat/dkim-b2
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12347 #MANAGER-12348
| License          | BSD 3-Clause

- [ ] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/modules/emailpro/src/domain/emailpro-domain.service.js [hunk 1]
```diff
@@ -207,6 +207,12 @@ export default /* @ngInject */ function EmailProDomains(
       .then(({ data }) => data);
   };
 
+  this.getDomain = function getDomain(organization, serviceName, domain) {
+    return this.services.$http
```
Comment (antonymarion) [inline packages/manager/modules/emailpro/src/domain/emailpro-domain.service.js:211]: image

Are you sure that the API is right here?

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js [hunk 2]
```diff
@@ -68,6 +68,16 @@ export default class EmailProDomainDkimAutoconfigCtrl extends DkimAutoConfigurat
       .finally(() => {
         this.loading.step1 = false;
       });
+
+    if (this.dkimStatus === this.GLOBAL_DKIM_STATUS.NOK) {
+      this.services.EmailProDomains.getDomain(
```
Comment (antonymarion) [inline packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js:73]: Is the api get in the service must be change, do not forget to remove the first parameter in the signature here

**Review by antonymarion — CHANGES_REQUESTED** [review]

**Review by antonymarion — APPROVED** [review]

File: packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js (hunk 2 above)
Comment (benjaminlepine) [reply packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.controller.js:73]: done

File: packages/manager/modules/emailpro/src/domain/emailpro-domain.service.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (benjaminlepine) [reply packages/manager/modules/emailpro/src/domain/emailpro-domain.service.js]: done

**Review by darsene — APPROVED** [review]

**Pull request merged**